model/system.go

```diff
@@ -23,6 +24,8 @@ const (
 	SYSTEM_WARN_METRIC_NUMBER_OF_ACTIVE_USERS_200 = "warn_metric_number_of_active_users_200"
 	SYSTEM_WARN_METRIC_NUMBER_OF_ACTIVE_USERS_400 = "warn_metric_number_of_active_users_400"
 	SYSTEM_WARN_METRIC_NUMBER_OF_ACTIVE_USERS_500 = "warn_metric_number_of_active_users_500"
+	AWS_METERING_REPORT_INTERVAL                  = 3
```

Comment:
This constants looks like is not used anywhere, I think it should be used to avoid magic numbers (I think now we are using `1*time.Hour` and I guess it should be `model.AWS_METERING_REPORT_INTERVAL*time.Hour`

Reply:
other option is to remove this constant.

Reply:
Yes, it is was intended to be used in the 1*time.Hour, got lost in some refactoring.